CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ just a few small guidelines you need to follow.
 
 1. File a bug at https://github.com/google/shaka-player/issues (if there isn't
    one already).  If your patch is going to be large, you should start a
-   discussion on the mailing list first to make sure it is in line with our
+   discussion on the [mailing list][] first to make sure it is in line with our
```

Comment:
I appreciate you trying to add a useful link, but this information is actually out of date.  We no longer have a discussion-type list, but instead ask for questions to be filed on GitHub issues.  The shaka-player-users list was changed a couple years ago into a list where only the team can post, for us to send out release announcements and developer surveys.

Reply:
Shall we update the docs by removing the statement instead. I think, it will bring more clarity.

Reply:
Yes, please.  You can say "start a discussion on GitHub first" instead.  Thank you!